src/libutil/file-content-address.hh

```diff
@@ -10,16 +10,28 @@ namespace nix {
 /**
  * An enumeration of the ways we can serialize file system
  * objects.
+ *
+ * See `file-system-object/content-address.md#serial` in the manual for
```

Comment:
thought:
I've linked to the `unstable` manual URL before in the external API docs, but we might want to weigh that differently for internal docs.